### DISCUSSION TITLE
Adding an option to retain read items for x days

### DISCUSSION THREAD

**Discussion by hexman1999** [spacecowboy/Feeder#236] [Ideas]
Feeder is my favorite rss reader but I have a problem with items that automatically removed just if I open it even if accidentally

So I suggest adding an option to retain topics for x days selected by user

And adding another separate option for removing read topics

Thanks

**Comment by spacecowboy** [comment]
Do you mean that the articles disappear because you'v opened them?

Just open the filter menu and toggle read/recently read items on there.

If you mean items being deleted, then it your per-feed limit is too low, just increase it.

**Reply by hexman1999** [reply]
Ok I found it there when putting filters

But there is no option to only show read items

And filter some feeds from all feeds tab for easier finding

**Reply by spacecowboy** [reply]
Just use tags
